### DISCUSSION TITLE
Active menu is not getting highlighted in horizontal menu.

### DISCUSSION THREAD

**Discussion by rahulAnand1999** [primefaces/community#3197] [PrimeReact Templates]
The active menu item is not being highlighted in the horizontal menu, which could be due to missing or incorrectly applied CSS styles, a failure to update the active class dynamically, or issues in the logic for detecting the current active route in your application. @melloware Please look into this issue.

Link 1- https://freya.primereact.org/uikit/formlayout
Link 2- https://atlantis.primereact.org/

**Comment by melloware** [comment]
I don't work for PrimeTek... So you will have to hope they see this...
